spring-ws-core/src/main/java/org/springframework/ws/server/endpoint/mapping/PayloadRootQNameEndpointMapping.java

```diff
@@ -43,9 +43,7 @@
  * @author Arjen Poutsma
  * @see org.springframework.xml.namespace.QNameEditor
  * @since 1.0.0
- * @deprecated as of Spring Web Services 2.0, in favor of {@link PayloadRootAnnotationMethodEndpointMapping}.
```

Comment:
Thank you @gregturn !
Although I think we should add `@see` from these classes to be flexible.

From other side recall me, please, if you did some changes to Docs on the matter.
Shouldn't we say anything there too?